### ISSUE TITLE
Suggestion : Automatic summaries (or table of contents) for wikis

### ISSUE THREAD

**Issue by Skyost** [isaacs/github#504]
Hi !     
This is just a suggestion, a bit like MediaWiki does :     
![Example](http://i.imgur.com/V2MTf7o.png)
I have done some research too and someone posted [this](https://gist.github.com/guilhermechapiewski/379469) but sadly, it does not work anymore.

Thank you !

**Comment by cirosantilli** [comment]
README: https://github.com/isaacs/github/issues/215

**Comment by cirosantilli** [comment]
Gollum's `[[_TOC_]]` seems disabled: https://github.com/cirosantilli/test-git-web-interface/wiki

**Comment by cirosantilli** [comment]
On SO: http://stackoverflow.com/questions/9239588/toc-or-sidebar-in-github-wiki

**Comment by mxmilkiib** [comment]
For those with *monkey browser extension, there is a user script to achieve this atm;

https://github.com/Mottie/GitHub-userscripts/wiki/GitHub-table-of-contents

**Comment by igorsantos07** [comment]
As per https://github.com/isaacs/github/issues/215#issuecomment-822666151 by @botatoes, the topic will be worked on......... someday.

**Comment by joric** [comment]
What's happening people? I saw the new feature and managed to delete all the contents and custom sidebars already and then it dissapeared. How to turn it back on?

![image](https://user-images.githubusercontent.com/852547/129961554-f960933f-4463-4a7d-a56f-cf83b356c6f4.png)

https://github.com/joric/nrfmicro/wiki

**Comment by joric** [comment]
Oh. It's back again. It's kind of not too good though. When there are a lot of pages, it's not too useful. Maybe move the current page in the sidebar on top. I'm reverting it for now.

![image](https://user-images.githubusercontent.com/852547/129973940-8aebffa0-17eb-4bf4-b180-a5487f66b013.png)

**Upd.** weirdly it's also case sensitive, e.g. https://github.com/joric/jorne/wiki/trackpoint vs https://github.com/joric/jorne/wiki/Trackpoint where's that feature discussion? Didn't find it in the github blog either.

**Comment by thorade** [comment]
https://twitter.com/github/status/1428466581554745352

**Comment by igorsantos07** [comment]
Yep, that kinda helps with finding what you want in the broad wiki, if you're into opening tree lists. Otherwise, it's still useless to navigate in the page you are currently viewing :shrug: Not to mention you must _want_ to open the "Pages" list if you have a sidebar committed.
